src/nixpacks/app.rs

```diff
@@ -55,10 +74,11 @@ impl App {
 mod tests {
     use std::collections::HashMap;
 
-    use super::*;
     use serde::{Deserialize, Serialize};
     use serde_json::{Map, Value};
 
+    use super::*;
+
```

Comment:
I don't think this changes anything, so perhaps this should be reverted to how it was?